Sources/Plasma/FeatureLib/pfPython/pyVaultAgeInfoNode.cpp

```diff
@@ -181,6 +181,10 @@ ST::string pyVaultAgeInfoNode::GetAgeFilename() const
 
 void pyVaultAgeInfoNode::SetAgeFilename(const ST::string& v)
 {
+    if (fNode) {
+        VaultAgeInfoNode access(fNode);
+        access.SetAgeFilename(v);
+    }
```

Comment:
Are any of the changes in this file relevant to the `findNode` search functionality, or are they just "while I've got this open, lemme just..." changes?

Reply:
Per the second commit message:
> Add setters to ptVaultAgeInfoNode.
>
> These are critically important to supporting the case of searching for arbitrary Ages in the player vault.

Reply:
Oh... I didn't notice the little `...` button that indicated the commit messages had more information. I apologize. Everything else seemed to make sense to me!